handlers/AdminHandlers/AdminGameHandlers.py

```diff
@@ -160,7 +160,7 @@ def post(self, *args, **kwargs):
             self.render("public/404.html")
 
     def add_source_code(self):
-        box = Box.by_uuid(self.get_argument('box_uuid'), '')
+        box = Box.by_uuid(self.get_argument('box_uuid'))
```

Comment:
You know what, I didn't even think to try that. I'll give it a go when I get home and see if I can reproduce the error with that.